gluten-core/src/main/scala/org/apache/gluten/backendsapi/SparkPlanExecApi.scala

```diff
@@ -419,7 +419,7 @@ trait SparkPlanExecApi {
       substraitExprName: String,
       child: ExpressionTransformer,
       original: Round): ExpressionTransformer = {
-    GenericExpressionTransformer(substraitExprName, Seq(child), original)
+    DecimalRoundTransformer(substraitExprName, child, original)
```

Comment:
It seems we can remove the API `genDecimalRoundTransformer` directly as it has the same implementation as Velox backend.

Reply:
Done